app/main.py

```diff
@@ -1,6 +1,22 @@
+import json
+import os
+from dotenv import load_dotenv
+
+import requests
+
+load_dotenv()
+
+BASE_URL = "https://api.weatherapi.com/v1/current.json?"
+API_KEY = os.environ["API_KEY"]
```

Comment:
Ensure that the environment variable 'API_KEY' is set before running the script. Consider adding error handling to check if the API key is missing.